### PR TITLE
Upgrade to Sylius 1.11

### LINKED ISSUE #119: Allow for Sylius 1.11

### PR DESCRIPTION
**Pull request by lruozzi9**
Fixes #119 

Changes proposed in this pull request:
- Drop support for Sylius 1.9 and 1.10
- Drop support for Symfony 4.4

@webgriffe/wg-devs what do you think about this PR :shipit:?

### CODE REVIEW

File: tests/Application/config/packages/security.yaml [hunk 1]
```diff
@@ -1,14 +1,3 @@
-parameters:
```
Comment (LucaGallinari) [inline tests/Application/config/packages/security.yaml:1]: Why all parameters have been removed?

File: tests/Application/config/packages/api_platform.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+api_platform:
```
Comment (LucaGallinari) [inline tests/Application/config/packages/api_platform.yaml:1]: This could not be required if we disable API as previous comment

File: tests/Application/config/packages/framework.yaml [hunk 3]
```diff
@@ -4,3 +4,6 @@ framework:
     csrf_protection: true
     session:
         handler_id: ~
+    serializer:
```
Comment (LucaGallinari) [inline tests/Application/config/packages/framework.yaml:7]: Why this config?

File: bin/create_node_symlink.php [hunk 4]
```diff
@@ -1,19 +1,45 @@
 <?php
```
Comment (LucaGallinari) [inline bin/create_node_symlink.php:1]: Why these changes?

**Review by LucaGallinari — CHANGES_REQUESTED** [review]
It would be useful to have the source URLs of the changes (if taken from somewhere) or a few words about it in the description of the PR (if custom), about the changes that have been made, otherwise it's hard to discern their correctness, or it's a double effort to find PRs etc in the web.
I found these:
- [Sylius skeleton plugin 1.11 bump](https://github.com/Sylius/PluginSkeleton/pull/318)
- [Fixes to security.yaml](https://github.com/Sylius/PluginSkeleton/pull/312)
but something is missing

**Comment by LucaGallinari** [comment]
Oh, ok. Found this: https://github.com/Sylius/PluginSkeleton/pull/321/files
Didn't see it as it has not been released yet, so the compare didn't find it.

File: tests/Application/config/packages/_sylius.yaml [hunk 5]
```diff
@@ -13,3 +13,6 @@ parameters:
 sylius_shop:
     product_grid:
         include_all_descendants: true
+
+sylius_api:
+    enabled: true
```
[the remark this answers is not in the document]
Comment (LucaGallinari) [reply tests/Application/config/packages/_sylius.yaml:18]: https://github.com/Sylius/PluginSkeleton/pull/321/files

File: tests/Application/config/packages/framework.yaml (hunk 3 above)
Comment (LucaGallinari) [reply tests/Application/config/packages/framework.yaml:7]: https://github.com/Sylius/PluginSkeleton/pull/321/files

File: bin/create_node_symlink.php (hunk 4 above)
Comment (LucaGallinari) [reply bin/create_node_symlink.php:1]: https://github.com/Sylius/PluginSkeleton/pull/321/files

File: tests/Application/config/packages/security.yaml (hunk 1 above)
Comment (LucaGallinari) [reply tests/Application/config/packages/security.yaml:1]: https://github.com/Sylius/PluginSkeleton/pull/321/files

File: bin/create_node_symlink.php (hunk 4 above)
Comment (lruozzi9) [reply bin/create_node_symlink.php:1]: It is now aligned with the Sylius plugin skeleton. I usually check for also not differed files to be sure that the plugin is maintained in sync with the skeleton.

**Comment by lruozzi9** [comment]
> Oh, ok. Found this: https://github.com/Sylius/PluginSkeleton/pull/321/files Didn't see it as it has not been released yet, so the compare didn't find it.

Yes, that PR has not been already tagged but is needed to make tests green and also to align the test/application with the Sylius 1.11 configurations.

**Comment by lruozzi9** [comment]
It still fails on JS behat, I think that is due again to the admin icon image 🤬

**Comment by lruozzi9** [comment]
I think this https://github.com/Sylius/Sylius/pull/13753 could resolve the problem definitively, but until is not released I don't know how to fix the build 🤷‍♂️

**Comment by lruozzi9** [comment]
Another problem is that composer does not install Sylius 1.11.2 (https://github.com/Sylius/PluginSkeleton/issues/323). The problem is still present in local also with that version.

**Review by LucaGallinari — APPROVED** [review]
